src/Dialect/ONNX/ONNXOps.td.inc

```diff
@@ -657,10 +664,12 @@ def ONNXConstantOp:ONNX_Op<"Constant",
   OpBuilderDAG<(ins "Attribute":$sparse_value, "Attribute":$value), [{
    if (value) {
     auto tensorType = value.getType();
-    build($_builder, $_state, tensorType, sparse_value, value);
+    build($_builder, $_state, tensorType, sparse_value, value,
+      FloatAttr(), ArrayAttr(), IntegerAttr(), ArrayAttr(), StringAttr(), ArrayAttr());
    } else {
     auto tensorType = sparse_value.getType();
-    build($_builder, $_state, tensorType, sparse_value, value);
+    build($_builder, $_state, tensorType, sparse_value, value,
+      FloatAttr(), ArrayAttr(), IntegerAttr(), ArrayAttr(), StringAttr(), ArrayAttr());
```

Comment:
@chentong319 Are the parameters after "value": these 6 attributes just null ptrs? If so, could we not have a custom builder where they are not needed as params?

Maybe a better question is when are they needed? From ONNX dag, there will be one that is used and the others are not?

This is just a small stylistic issue, so no worry